profiling/tests/form.rs

```diff
@@ -16,7 +16,11 @@ fn open<P: AsRef<Path>>(path: P) -> Result<Vec<u8>, Box<dyn Error>> {
     Ok(buffer)
 }
 
-fn multipart(exporter: &ProfileExporter, internal_metadata: Option<serde_json::Value>) -> Request {
+fn multipart(
+    exporter: &ProfileExporter,
+    internal_metadata: Option<serde_json::Value>,
+    system_info: Option<serde_json::Value>,
```

Comment:
Maybe passing the `internal_metadata` and `system_info` as [`Box<serde_json::RawValue>`](https://docs.rs/serde_json/latest/serde_json/value/struct.RawValue.html) would be more efficient, because you can keep the jsons in String format.

Reply:
Probably. But it also feels like it may hide issues with json definitions in test a bit further down in the stacktrace rather than at the json definition part. Since this only applies for tests where efficiency is not such a priority imo, I decided to leave as is.